### PR TITLE
DT-831: default mode is project

### PR DESCRIPTION
**Pull request by zyuzka**

### CODE REVIEW

**Comment by dereuromark** [comment]
Best to use an input field that has some auto-select for the existing namespaces. 
This way you can easily use the defaults, and also easily exchange with any custom project or core namespace.

File: .travis.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ matrix:
     fast_finish: true
 
 before_script:
+  - composer install
```
Comment (dereuromark) [inline .travis.yml]: why is this needed?
it will basically do update here then anyway, as install without look file does this.

File: .travis.yml [hunk 2]
```diff
@@ -21,6 +21,7 @@ matrix:
...
   - phpenv config-add ci/travis.php.ini
   - composer remove --dev spryker/git-hook
```
Comment (dereuromark) [inline .travis.yml]: why do we need this? we should remove the whole thing for the repo IMO.

Comment (zyuzka) [reply .travis.yml]: Composer couldn' remove spryker/git-hook without install command

Comment (dereuromark) [reply .travis.yml]: i would also remove the whole git-hook thing from core modules.

Comment (zyuzka) [reply .travis.yml]: Got it

File: .travis.yml [hunk 3]
```diff
@@ -22,7 +22,6 @@ matrix:
 
 before_script:
   - phpenv config-add ci/travis.php.ini
-  - composer remove --dev spryker/git-hook
```
Comment (dereuromark) [inline .travis.yml]: this is only valid if you also remove the source of the problem.. https://github.com/spryker-sdk/spryk/blob/master/composer.json#L24

Comment (dereuromark) [reply .travis.yml]: if tests pass, then never mind.
